sql/item_equipment.sql

```diff
@@ -11582,6 +11582,8 @@ INSERT INTO `item_equipment` VALUES (23793,'ziamet_peti',1,0,4194303,172,0,0,32,
 INSERT INTO `item_equipment` VALUES (23794,'ziamet_bazubands',1,0,4194303,172,0,0,64,0,0);
 INSERT INTO `item_equipment` VALUES (23795,'ziamet_salvars',1,0,4194303,172,0,0,128,0,0);
 INSERT INTO `item_equipment` VALUES (23796,'ziamet_nails',1,0,4194303,172,0,0,256,0,0);
+INSERT INTO `item_equipment` VALUES (23803,'poroggo_cassock',1,0,4194303,397,0,0,32,16,0); -- TODO: capture model from retail
+INSERT INTO `item_equipment` VALUES (23804,'poroggo_cass._+1',1,0,4194303,397,0,0,32,16,0); -- TODO: capture model from retail
```

Comment:
Nope. Its a blue frog suit. If you can't capture it, there is a captures discord this a request can be made from. Or you can use the GM command to set model till you find it. but I got fed up with half implemented items forever ago.

Reply:
someone should probably update all the other "TODO: capture model from retail" So people stop adding it LOL

Reply:
Grandfathered in until fix, but I put a stop to new ones burying the project alive, LOL

Reply:
I don't see a GM command for this? what is it? LOL
I'll get right on the rest of the TODOs if the command exists!

Reply:
oh I'm dumb I found it

Reply:
Blue frog all set 👍